src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/framework/checker/CheckerTypes.kt

```diff
@@ -48,6 +38,28 @@ internal interface CheckerContext {
     fun checkDefaultRawReport(mustExist: Boolean = true)
 }
 
+/**
+ * Static info about Gradle project.
+ */
+internal interface ProjectAnalyze {
```

Comment:
'Analyze' is a verb, so maybe you'd be better of with 'ProjectAnalysisResult' or just 'ProjectInfo'/'ProjectDescription'